jsconfig.json

```diff
@@ -14,7 +14,8 @@
     },
     "target": "es5",
     "noEmit": true,
-    "skipLibCheck": false
+    "skipLibCheck": false,
+    "types": ["vitest/globals"]
```

Comment:
Was this not an issue for everyone else btw? In every single file (beyond our TS tests, which set this in their `tsconfig.json` files) I get hundreds of `Cannot fine name 'describe'/'expect'. Do you need to install type definitions...`

I'd almost have to assume folks are using other editor setups than me as it makes mine light up like a christmas tree without this